src/kOS/Communication/RunCommand.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using kOS.Safe.Compilation;
+
+namespace kOS.Communication
+{
+    public class RunCommand : InterProcCommand
+    {
+        public List<Opcode> Program;
+
+        public override void Execute(SharedObjects shared)
+        {
+            if (shared.Cpu != null)
+            {
+                shared.Cpu.RunProgram(Program);
+            }
+        }
+    }
+}
```

Comment:
I'm not sure if we want to keep the logic. I think @Dunbaratu explicitly asked me to remove this. I simply wanted to ask for the final confirmation later and push a separate PR if needed.